numpy/core/tests/test_multiarray.py

```diff
@@ -1812,65 +1850,6 @@ def test_sort(self):
             c.sort(kind=kind)
             assert_equal(c, a, msg)
 
-        # test datetime64 sorts.
-        a = np.arange(0, 101, dtype='datetime64[D]')
-        b = a[::-1]
-        for kind in ['q', 'h', 'm']:
-            msg = "datetime64 sort, kind=%s" % kind
-            c = a.copy()
-            c.sort(kind=kind)
-            assert_equal(c, a, msg)
-            c = b.copy()
-            c.sort(kind=kind)
-            assert_equal(c, a, msg)
-
-        # test timedelta64 sorts.
-        a = np.arange(0, 101, dtype='timedelta64[D]')
-        b = a[::-1]
-        for kind in ['q', 'h', 'm']:
-            msg = "timedelta64 sort, kind=%s" % kind
-            c = a.copy()
-            c.sort(kind=kind)
-            assert_equal(c, a, msg)
-            c = b.copy()
-            c.sort(kind=kind)
-            assert_equal(c, a, msg)
-
-        # check axis handling. This should be the same for all type
-        # specific sorts, so we only check it for one type and one kind
-        a = np.array([[3, 2], [1, 0]])
-        b = np.array([[1, 0], [3, 2]])
-        c = np.array([[2, 3], [0, 1]])
-        d = a.copy()
-        d.sort(axis=0)
-        assert_equal(d, b, "test sort with axis=0")
-        d = a.copy()
-        d.sort(axis=1)
-        assert_equal(d, c, "test sort with axis=1")
-        d = a.copy()
-        d.sort()
-        assert_equal(d, c, "test sort with default axis")
-
-        # check axis handling for multidimensional empty arrays
-        a = np.array([])
-        a.shape = (3, 2, 1, 0)
-        for axis in range(-a.ndim, a.ndim):
-            msg = 'test empty array sort with axis={0}'.format(axis)
-            assert_equal(np.sort(a, axis=axis), a, msg)
-        msg = 'test empty array sort with axis=None'
-        assert_equal(np.sort(a, axis=None), a.ravel(), msg)
-
-        # test generic class with bogus ordering,
-        # should not segfault.
-        class Boom:
-            def __lt__(self, other):
-                return True
-
-        a = np.array([Boom()]*100, dtype=object)
-        for kind in self.sort_kinds:
-            msg = "bogus comparison object sort, kind=%s" % kind
-            c.sort(kind=kind)
```

Comment:
To reduce the size of the diff, can you leave these tests at the bottom, perhaps by adding:
```python
def test_sort_axis():
    # check axis handling.
    ...

def test_sort_size_0(self):
    # check axis handling for multidimensional empty arrays
    ...

def test_sort_bad_ordering(self):
    # test generic class with bogus ordering,
    ...
```